src/misc.go

```diff
@@ -501,7 +501,7 @@ var regExtStrMap = map[string]string{
 
 	"docx?": "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
 	"pptx?": "application/vnd.ms-powerpoint",
-	"tsv":   "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
```

Comment:
Cool, nope 'text/tab-separated-values' just looked odd in comparison to 'text/csv'.